Sources/Basics/JSON+Extensions.swift

```diff
@@ -89,8 +89,9 @@ extension JSONEncoder {
             } else {
                 encoder.outputFormatting = [.prettyPrinted]
             }
-            #elseif compiler(>=5.3)
-            encoder.outputFormatting = [.sortedKeys, .prettyPrinted, .withoutEscapingSlashes]
+            // `.withoutEscapingSlashes` is not in 5.3 on non-Darwin platforms
```

Comment:
Suggested copy-edit:

```suggestion
            // `.withoutEscapingSlashes` is available in Swift 5.3 only on Darwin platforms
```